graphistry/embed_utils.py

```diff
@@ -571,7 +550,14 @@ def __len__(self) -> int:
         return self.num_steps
 
     def __getitem__(self, i:int):
-        _, torch, nn, dgl, GraphDataLoader, _, F, _ = lazy_embed_import_dep()
+        torch = deps.torch
+        if torch:
```

Comment:
i don't think the `if` is necessary? these codes will fail without the imports , and there are no fallbacks here, so just run without  `if`